src/components/ActivityFeed/ActivityFeedPostReactions.tsx

```
@@ -53,17 +53,21 @@ export const ActivityFeedPostReactions: React.FC<{
 			<div className={classNames('max-h-24 w-min overflow-y-auto pt-1', dmSans.className, dmSans.variable)}>
-				{usernames?.map((name: string, index: number) => (
+				{usernames?.slice(1)?.map((name: string, index: number) => (
 					<Link
 						href={`https://${network}.polkassembly.io/user/${name}`}
 						key={index}
 						target='_blank'
 						className='mb-[6px] flex items-center gap-[6px]'
 					>
 						<ImageComponent
-							src={userImages && userImages[index] && userImages[index]}
+							src={userImages && userImages.slice(1)?.length ? userImages.slice(1)[index] : undefined}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)